android/library/src/test/java/io/logto/android/client/LogtoAndroidClientTest.kt

```diff
@@ -60,57 +65,68 @@ class LogtoAndroidClientTest {
 
     @Test
     fun getOidcConfigurationAsyncShouldCallBlock() = runBlockingTest {
-        doReturn(oidcConfigurationMock).`when`(logtoAndroidClientSpy).getOidcConfiguration()
-        val handleOidcConfigurationCallbackMock: HandleOidcConfigurationCallback = mock()
+        coEvery {
+            logtoAndroidClientSpy.getOidcConfiguration()
+        } returns oidcConfigurationMock
+        val handleOidcConfigurationCallbackMock: HandleOidcConfigurationCallback =
+            mockk(relaxed = true)
 
         logtoAndroidClientSpy.getOidcConfigurationAsync(handleOidcConfigurationCallbackMock)
             .invokeOnCompletion { throwable ->
                 assertThat(throwable).isNull()
-                verify(handleOidcConfigurationCallbackMock)
-                    .invoke(eq(null), eq(oidcConfigurationMock))
+
+                val logtoExceptionMutableList = mutableListOf<LogtoException?>()
```

Comment:
call it `logtoExceptionSlot` would be better

Reply:
rename to `*CaptureList`